docs/contributing.md

```diff
@@ -0,0 +1,27 @@
+Contributing to Fluo
+====================
+
+Building Fluo
+-------------
+
+If you have [Git], [Maven], and [Java](version 7+) installed, run these commands
+to build Fluo:
+
+    git clone https://github.com/fluo-io/fluo.git
+    cd fluo
+    mvn package
+
+Testing Fluo
+------------
+
+Fluo has a test suite that consists of the following:
+
+* Units tests which are run by `mvn package`
+* Integration tests which are run using `mvn verify`.  These tests start
+a local Fluo instance (called MiniFluo) and run against it.
+* A [Stress test][Stress] application designed to run on a Fluo cluster.
```

Comment:
Opinionated comment, feel free to ignore. I would replace 'Fluo cluster' with just 'cluster'.